main/tests/cypress/cypress/e2e/extensions/wikibase/select_wikibase_instance.cy.js

```diff
@@ -80,6 +80,7 @@ describe(__filename, function () {
                 root: 'https://or-wikibase-test.wiki.opencura.com/wiki/',
                 main_page: 'https://or-wikibase-test.wiki.opencura.com/wiki/Main_Page',
                 api: 'https://or-wikibase-test.wiki.opencura.com/w/api.php',
+                mandatoryMediaInfoPropertyIds: ["P7482", "P170", "P6216"]
```

Comment:
I think we can also remove this line, since it is specific to Wikimedia Commons.